classes/class.ilAdobeConnectConfigGUI.php

```diff
@@ -672,6 +673,33 @@ public function initIliasSettingsForm()
 		$matrix->setHtml($tbl);
 		
 		$this->form->addItem($matrix);
+		
+		$api_version = new ilNonEditableValueGUI($this->pluginObj->txt('api_version'), 'api_version');
+		$this->form->addItem($api_version);
+		
+		$html_client = new ilCheckboxInputGUI($this->pluginObj->txt('html_client'), 'html_client');
+		$html_client->setInfo($this->pluginObj->txt('html_client_info'));
+		if(version_compare(ilAdobeConnectServer::getSetting('api_version'), '10.0.0' ,'<'))
+		{
+			$html_client->setDisabled(true);
+		}
+		$this->form->addItem($html_client);
+	}
+
+	/**
+	 * 
+	 */
+	private function readApiVersion()
+	{
+		$xmlAPI = ilXMLApiFactory::getApiByAuthMode();
```

Comment:
Does this also work even if the plugin configuration is not complete? We should prevent an issue/error if the settings view is accessed by the user, but the configuration is not complete, yet. The settings should not be displayed for an incomplete configuration.